foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -2767,9 +2842,9 @@ public enum AuthenticationState {
 }
 ```
 
-**VisitorID**
+**VisitorID**<a id="android-visitorid"></a>
 
-This class is an identifier to be used with the Experience Cloud Visitor ID Service.
+This class is an identifier to be used with the Adobe Experience Cloud Identity Service.
```

Comment:
Same here, the full name was mentioned before on this page, so let's use simple ECID Service - same below.

Reply:
As this page is a reference, I don't expect users to read through it like a book. I've been taking the approach of using the full name on first mention in a section. In this case, the first mention is in a tab within a section, so I'm using the full name in each tab with the reasoning that a user is looking at one platform at a time.

Reply:
Got it. In my opinion we should keep the text easy/quick to read, so let's think about how can we simplify this in tabs also. I am ok with keeping it as is for now and we can resolve this in a separate PR if needed.